<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublisherMenu</class>
 <widget class="QDialog" name="PublisherMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>274</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Pub_Connect">
        <property name="text">
         <string>Connect to Mqtt Broker</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_Static_Pub_Message">
      <property name="text">
       <string>Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Static_Pub_Topic">
      <property name="text">
       <string>Topic:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Pub_Topic"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit_Pub_Message"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Pub_Publish">
      <property name="text">
       <string>Publish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
